.github/workflows/conda.yml

```diff
@@ -48,36 +58,53 @@ jobs:
 
       - uses: conda-incubator/setup-miniconda@v2
         name: Vanilla install from conda
-        if: ${{ matrix.cfg.openeye == 'false' }}
         with:
           python-version: ${{ matrix.cfg.python-version }}
           activate-environment: test
           environment-file: devtools/conda-envs/conda.yaml
           auto-activate-base: false
-      - uses: conda-incubator/setup-miniconda@v2
-        name: Install from conda with OpenEye
-        if: ${{ matrix.cfg.openeye == 'true' }}
-        with:
-          python-version: ${{ matrix.cfg.python-version }}
-          activate-environment: test
-          environment-file: devtools/conda-envs/conda_oe.yaml
-          auto-activate-base: false
 
-      - name: Additional info about the build
+      - name: Environment with vanilla install
         shell: bash -l {0}
         run: |
-          uname -a
-          df -h
-          ulimit -a
+          conda info
+          conda list
+
+      - name: Import core classes
+        shell: bash -l {0}
+        run: |
+          which python
+          python -c "from openff.toolkit.topology import Molecule, Topology"
+          python -c "from openff.toolkit.typing.engines.smirnoff import ForceField"
+          python -c "from openff.toolkit.utils import *"
+
+      - name: Install test dependencies
+        shell: bash -l {0}
+        run: |
+          conda install openmmforcefields pytest nbval -c conda-forge
```

Comment:
I think we should remove openmmforcefields, since it's only used in the examples, and those are tested in a different CI build now (#911). 

I had passed over this as unimportant at first, but I think this could actually be risky --  this could cover up a deployment issue, since `openff-toolkit` is a dep of `openmmforcefields`